api/src/main/java/org/pmiops/workbench/interceptors/AuthInterceptor.java

```diff
@@ -110,16 +114,15 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
     }
 
     final String token = authorizationHeader.substring("Bearer".length()).trim();
-    final Userinfoplus OAuth2Userinfo = userInfoService.getUserInfo(token);
+    final Userinfoplus userInfo = userInfoService.getUserInfo(token);
```

Comment:
Opportunistic rename – the original var didn't conform to style guidelines, and userInfo is sufficiently precise.

Reply:
nit: `Userinfoplus` isn't the most descriptive class name. Maybe `oauthInfo`? "user" is implied I think.

Reply:
I agree, but we don't control that class. It's part of Google's OAuth2 implementation.

Reply:
I understand,  but I'm saying  changing the var  name to  be more descriptive at the  expense of matching the type would help.